hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/x509/exception/CertificateException.java

```diff
@@ -88,7 +88,6 @@ public enum ErrorCode {
     BOOTSTRAP_ERROR,
     CSR_ERROR,
     CRYPTO_SIGNATURE_VERIFICATION_ERROR,
-    CERTIFICATE_NOT_FOUND_ERROR,
```

Comment:
should we delete unused enums? The cardinality changes.. should be harmless but still.

Reply:
About this one I am pretty sure it does not go through any protocol or conversion during runtime.
We have a couple other similar ErrorCode enums for different exceptions, I left those as they are for now, because I felt that if I want to be sure then I need a bit more involved lookup for how the exception is used.